Manager/src/main/java/io/raspberrywallet/manager/modules/exceptions/KeypartDecryptionException.java

```diff
@@ -0,0 +1,31 @@
+package io.raspberrywallet.manager.modules.exceptions;
+
+public class KeypartDecryptionException extends Throwable {
```

Comment:
could you merge it with DecryptionException class as I did in my branch?

https://github.com/RaspberryWallet/Backend/blob/module-authorizationserver/Manager/src/main/java/io/raspberrywallet/manager/cryptography/crypto/exceptions/DecryptionException.java